packages/contentstack-export/src/commands/cm/stacks/export.js

```diff
@@ -21,7 +21,7 @@ class ExportCommand extends Command {
     const securedAssets = exportCommandFlags['secured-assets'];
     const data = exportCommandFlags.data || exportCommandFlags['data-dir'];
     const moduleName = exportCommandFlags.module;
-    const contentTypes = exportCommandFlags['content-type'];
+    const contentTypes = exportCommandFlags['content-types'];
```

Comment:
This is invalid as we are passing single CT in export.

Reply:
Content types in export are multiple so keeping this change